students/17th/team3/hojinjenechoi/user/views.py

```diff
@@ -0,0 +1,55 @@
+import json
+
+from django.views           import View
+from django.http            import JsonResponse 
+
+from user.models  import User
+
```

Comment:
```suggestion
MINIMUM_PASSWORD_LENGTH = 8
```
상수값은 클래스 밖이나 , 함수 바깥에 위치 시켜주세요. 재사용 범위에 따라 결정하시면 됩니다.

Reply:
넵 마지막으로 수정하겠습니다! 감사합니다!!